InvenTree/label/admin.py

```diff
@@ -1,15 +1,87 @@
 """Admin functionality for the 'label' app"""
 
+from django import forms
 from django.contrib import admin
+from django.utils.translation import gettext_lazy as _
 
-from .models import PartLabel, StockItemLabel, StockLocationLabel
+from .models import (LabelTemplate, PartLabel, StockItemLabel,
+                     StockLocationLabel)
+
+
+class LabelAdminForm(forms.ModelForm):
+    """Custom form for the label's admin form to inject non-model fields"""
+    class Meta:
+        """Meta is necessary to set a more reasonable field order"""
+        model = LabelTemplate
+        fields = ['metadata', 'name', 'description', 'label', 'enabled', 'width', 'height',
+                  'filename_pattern', 'multipage', 'pagesize_preset', 'page_orientation',
+                  'page_width', 'page_height', 'multipage_border']
+
+    pagesize_preset = forms.ChoiceField(
+        choices=[
+            ('custom', _('Custom')),
+            ('210x297', _('A4')),
+            ('297x420', _('A3')),
+            ('148x210', _('A5')),
+        ],
+        label=_('Preset paper size'),
```

Comment:
Question: I am using metric but does it make sense to add 1 or 2 common American sheet sizes?

Reply:
Sure we can, what do you think which one would make the most sense? Letter+legal?

Reply:
I am EU-based and have never used American formats but a quick search suggests letter, legal and maybe tabloid might make sense.

Reply:
Certainly letter and legal would be good options to add in.